src/annotator/adder.js

```diff
@@ -192,6 +195,9 @@ export class Adder {
     // and close to the end.
     const hMargin = Math.min(ARROW_H_MARGIN, selectionRect.width);
     const adderWidth = this._width();
+    // Render the adder a little lower on a touch device to provide room for the
+    // selection grab handles.
```

Comment:
By this comment you mean that we're moving the adder lower so that interactions with it are not confused with interactions with the selection grab handles?

Reply:
correct. I will re-word that a bit